pages/management/PKPToolsHandler.inc.php

```diff
@@ -160,4 +166,101 @@ public function resetPermissions($args, $request)
         echo json_encode(true);
         die;
     }
+
+    public function jobs($args, $request)
+    {
+        $this->setupTemplate($request, true);
+
+        $templateMgr = TemplateManager::getManager($request);
+
+        $queuedJobs = $this->buildQueuedJobsTable();
+
+        $totalQueuedJobs = $queuedJobs['total'];
+        $queuedData = $queuedJobs['data'];
+
+        $state = [
+            'components' => [
+                $queuedData->id => $queuedData->getConfig(),
```

Comment:
This is more @NateWr's area of expertise -- but isn't `id` a string, and `getConfig` returns an array?

Reply:
There's a few other examples using this methodology.

https://github.com/pkp/pkp-lib/blob/main/pages/dashboard/DashboardHandler.inc.php#L188

Reply:
(reminder about this comment!)

Reply:
He's getting the result from `id` parameter and using as the key value.

Something like this:
![Screenshot from 2022-01-19 12-06-50](https://user-images.githubusercontent.com/1243634/150157832-316eb0ee-334e-481d-8ec0-7b465b54f060.png)

Reply:
I kept misreading...
```php
$queuedData->id => $queuedData->getConfig()
```
...as...
```php
$queuedData->id = $queuedData->getConfig()
```
My mistake!